dist/components/tabs/_tabs.scss

```diff
@@ -0,0 +1,117 @@
+// Tabs
+// ====
+//
+// Tabs component with tabs controls and tabs body.
+// Horizontal controls tabs items placed in one line.
+
+
+// Configurable Variables
+// ---
+
+$tabs__item-size: 150px !default;
+$tabs__padding: 10px 5px !default;
+$tabs__body-space: 20px !default;
+$tabs__active-background: hotpink;
```

Comment:
Agreed, we should follow the same default conventions as in our other stencil components, like [here](https://github.com/mobify/stencil/blob/master/dist/components/alert/_alert.scss#L10)